e2e-tests/tests/topics.spec.ts

```diff
@@ -8,6 +10,48 @@ test('Go to topic page', async ({ context }) => {
   await page.getByRole('link', { name: 'Rosh Hashanah' }).first().isVisible();
 });
 
+test('Check source', async ({ context }) => {
+  const page = await goToPageWithLang(context, '/topics');
+  await page.getByRole('link', { name: 'Jewish Calendar' }).click();
+  await page.getByRole('link', { name: 'Shabbat' }).first().click();
+  await page.getByRole('link', { name: 'Notable Sources' }).first().isVisible();
+  await page.getByRole('link', { name: 'All Sources' }).first().isVisible();
+});
+
+test('Check admin tab', async ({ context }) => {
+  const page = await goToPageWithUser(context, '/topics', testAdminUser);
+  await page.getByRole('link', { name: 'Jewish Calendar' }).click();
+  await page.getByRole('link', { name: 'Shabbat' }).first().click();
+  await page.getByRole('link', { name: 'Notable Sources' }).first().isVisible();
+  await page.getByRole('link', { name: 'All Sources' }).first().isVisible();
+  await page.getByRole('link', { name: 'Admin' }).first().isVisible();
+});
+
+test('Check sources he interface', async ({ context }) => {
+  const page = await goToPageWithLang(context, '/topics', LANGUAGES.HE);
+  await page.getByRole('link', { name: 'מועדי השנה' }).click();
+  await page.getByRole('link', { name: 'שבת' }).first().click();
+  await page.getByRole('link', { name: 'מקורות מרכזיים' }).first().isVisible();
+  await page.getByRole('link', { name: 'כל המקורות' }).first().isVisible();
+});
+
+test('Check author page', async ({ context }) => {
+  const page = await goToPageWithLang(context, '/topics/jonathan-sacks');
+  await page.getByRole('link', { name: 'Works on Sefaria' }).first().isVisible();
+});
+
+test('Check redirection for sourceless topic', async ({ context }) => {
+  const page = await goToPageWithLang(context, '/topics/Monkey');
+  const expectedUrl = 'search?q=Monkey&tab=sheet&tvar=1&tsort=relevance&stopics_enFilters=Monkey&svar=1&ssort=relevance';
+  await page.waitForTimeout(10000)
```

Comment:
We should try to avoid timeouts at all costs as they either make the tests brittle (when too short) and greatly increase the run time (when too long). Is there no better way to do this? I think you can await an element on the search page to exist.